aiven/client/argx.py

```diff
@@ -33,6 +33,22 @@ def default(self, o):  # pylint:disable=E0202
         return jsonlib.JSONEncoder.default(self, o)
 
 
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescriptionHelpFormatter):
+    """Help formatter to display the default value only for integers and non-empty strings"""
+    def _get_help_string(self, action):
+        help_text = action.help
+        if '%(default)' not in action.help:
+            if action.default is not argparse.SUPPRESS:
+                defaulting_nargs = [argparse.OPTIONAL, argparse.ZERO_OR_MORE]
```

Comment:
What's `defaulting`? Could just be inline in the `if` statement below?

Reply:
Never mind, misread that and understood that it's a default for `nargs`, not if a default help text should be used for certain `nargs` values.